pkgs/applications/misc/fspy/default.nix

```
@@ -19,6 +19,7 @@ in appimageTools.wrapType2 {
     description = "A cross platform app for quick and easy still image camera matching";
     license = licenses.gpl3;
     homepage = "https://fspy.io/";
+    knownVulnerabilities = [ "Vendors ancient Electron" ];
```

```suggestion
    knownVulnerabilities = [ "Vendors Electron 8 (end-of-life)" ];
```
to give some clarity